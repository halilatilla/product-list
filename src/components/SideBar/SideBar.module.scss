.products {
  display: grid;
  grid-template-columns: max-content 1fr;
  gap: 24px;
}
