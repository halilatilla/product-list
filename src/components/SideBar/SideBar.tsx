import { FC } from 'react'
import classnames from 'classnames'

import styles from './SideBar.module.scss'

interface Props {
  className?: string
}

const SideBar: FC<Props> = ({ className, ...rest }) => {
  return (
    <div className={classnames(styles.sideBar, className)} {...rest}>
      sideBar
    </div>
  )
}

export default SideBar
