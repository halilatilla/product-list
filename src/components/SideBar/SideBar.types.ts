import { IProduct } from '@src/types'

export default interface ISideBar {
  className?: string
  products: IProduct[]
}
