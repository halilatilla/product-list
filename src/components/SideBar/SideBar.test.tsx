import { render } from '@src/testUtils'

import SideBar from './SideBar'

describe('SideBar', () => {
  const PROPS = {
    products: [],
  }

  it.only('renders correctly', () => {
    const { container } = render(<SideBar {...PROPS} />)
    expect(container).toMatchSnapshot()
  })
})
