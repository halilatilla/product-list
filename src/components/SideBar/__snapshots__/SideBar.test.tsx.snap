// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideBar renders correctly 1`] = `
<div>
  <div
    class="sidebar"
  >
    <div
      class="filter"
    >
      <ul />
    </div>
    <div
      class="filter"
    >
      <p
        class="title"
      >
        sıralama
      </p>
      <ul>
        <li>
          <button
            class="button flex-center text item"
            type="button"
          >
            en düşük fiyat
             
          </button>
        </li>
        <li>
          <button
            class="button flex-center text item"
            type="button"
          >
            en yeniler (A-Z)
             
          </button>
        </li>
        <li>
          <button
            class="button flex-center text item"
            type="button"
          >
            en yeniler (Z-A)
             
          </button>
        </li>
        <li>
          <button
            class="button flex-center text item"
            type="button"
          >
            en yüksek fiyat
             
          </button>
        </li>
      </ul>
    </div>
    <div
      class="filter"
    >
      <ul />
    </div>
  </div>
</div>
`;
